spec/requests/post_index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature 'PostIndex', type: :feature do
+  it 'response body includes correct placeholder text' do
+    visit posts_index_path
+    expect(page).to have_content('Lists of all posts')
```

Comment:
Also kindly test whether the correct template is being rendered by using eg., `expect(response).to render_template(:show)`